machineconfiguration/v1alpha1/0000_80_machineconfignode-CustomNoUpgrade.crd.yaml

```diff
@@ -0,0 +1,199 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: machineconfignodes.machineconfiguration.openshift.io
+  labels:
+    "openshift.io/operator-managed": ""
+  annotations:
+    api-approved.openshift.io: https://github.com/openshift/api/pull/1596
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    release.openshift.io/feature-set: CustomNoUpgrade
+spec:
+  # group name to use for REST API: /apis/<group>/<version>
+  group: machineconfiguration.openshift.io
+  scope: Cluster
+  names:
+    kind: MachineConfigNode
+    singular: machineconfignode
+    plural: machineconfignodes
+  versions:
+    - name: v1alpha1
+      # Each version can be enabled/disabled by Served flag.
+      served: true
+      # One and only one version must be marked as the storage version.
+      storage: true
+      subresources:
+        status: {}
+      additionalPrinterColumns:
+        - jsonPath: .status.conditions[?(@.type=="Updated")].status
+          name: Updated
+          type: string
+        - jsonPath: .status.conditions[?(@.type=="UpdatePrepared")].status
+          name: UpdatePrepared
+          type: string
+        - jsonPath: .status.conditions[?(@.type=="UpdateExecuted")].status
```

Comment:
What is the difference between this condition and `Updated`?

Reply:
UpdateExecuted indicates that the core of the update has occured. The core of an update is the step in between UpdatePrepared and UpdatePostActionCompleted. It is a step in the process being viewed in the past tense as others are so that I can use the T/F/Unknown patter.